corehq/ex-submodules/casexml/apps/phone/models.py

```diff
@@ -382,16 +382,6 @@ def delete_synclog(synclog_id):
             synclog.delete()
             deleted = True
     except ValidationError:
-        pass
-
-    try:
-        synclog = properly_wrap_sync_log(SyncLog.get_db().get(synclog_id))
-        SyncLog.get_db().delete_doc(synclog)
-        deleted = True
-    except ResourceNotFound:
-        pass
-
-    if not deleted:
```

Comment:
shouldn't this `if` still he here?

Reply:
No, I was using it to track couch vs sql deletion. If there were no synclog to delete, this should raise a missingsynclog exception.

Reply:
Actually fixed this as well